doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -1,18 +1,605 @@
+    To reset an Excel file, you simply set the flag `overwrite_file` as True:
+
+
+- code: |
+    arr = ndtest((3, 3))
+    
+    # 1. reset the file Excel file 'arrays.xlsx' (the sheet 'array_2D' is removed)
+    # 2. save the array arr in the sheet 'array_3D'
+    arr.to_excel('arrays.xlsx', 'array_3D', overwrite_file=True)
+
+  id: 2
+
+- markdown: |
+    ##### Specifying Position in Sheet
+    
+    By default, array are dumped starting at cell 'A1'. Using the argument ``position`` it is possible to change the top left cell of the dumped array. This can be useful when several arrays must be dumped in the same sheet:
+
+
+- code: |
+    arr2 = ndtest((2, 2))
+    arr3 = ndtest((3, 3))
+    
+    # save the arrays arr2 and arr3 in the same sheet 'two_arrays'
+    # the argument 'position' is used to shift the location of the second array to be dumped
+    arr2.to_excel('arrays.xlsx', 'two_arrays_same_sheet')
+    arr3.to_excel('arrays.xlsx', 'two_arrays_same_sheet', position='A5')
+
+  id: 3
+
+- markdown: |
+    ##### Specifying Wide VS Narrow format
+    
+    By default arrays are dumped in a ``wide`` format, meaning they are exported with the last axis represented horizontally. For exmaple, a 2D array will be exported as:
+    
+    | a\b | b0  | b1  |
+    | --- | --- | --- |
+    | a0  | 0   | 1   |
+    | a1  | 2   | 3   |
+    
+    By setting the argument ``wide`` to False, arrays are exported in a ``narrow`` format: one column per axis plus one value column. For example:
+    
+    | a   | b   | value |
+    | --- | --- | ----- |
+    | a0  | b0  | 0     |
+    | a0  | b1  | 1     |
+    | a1  | b0  | 2     |
+    | a1  | b1  | 3     |
+    
+    The name of the column containing the data can changed by setting the argument ``value_name``.
+
+
+- code: |
+    arr = ndtest((2, 2))
+    
+    # dump the array arr in a narrow format (one column per axis plus one value column)
+    # by default, the name of the column containing data is set to 'value'
+    arr.to_excel('arrays.xlsx', 'array_narrow', wide=False)
+    
+    # dump the array arr in a narrow format and set the name of the column 
+    # containing data to 'data'
+    arr.to_excel('arrays.xlsx', 'array_narrow', wide=False, value_name='data')
+
+  id: 4
+
+- markdown: |
+    ##### Exporting only data
+    
+    By setting the argument ``header`` to False, only the data are exported while axes names and labels are dropped.
+
+
+- code: |
+    arr = ndtest((2, 2))
+    
+    # drop axes names and labels. Only data are exported
+    arr.to_excel('arrays.xlsx', 'data_only', header=False)
+
+  id: 5
+
+- markdown: |
+    #### Dumping Arrays to Excel Sheets with open_excel 
+
+
+- markdown: |
+    ##### Basic Usage
+    
+    ``open_excel`` must be used with the Python keyword ``with``. If the Excel file doesn't exist, the argument ``overwrite_file`` must set to True. The method ``save`` must be called to actually write data in the Excel file. 
```

Comment:
s/must be/should be/?
s/``with``/``with``, which ensures that the file is properly closed even if an error occurs/